### ISSUE TITLE
Support `memo` when transacting assets

### ISSUE THREAD

**Issue by 3Nigma** [the-creators-galaxy/hts-distribution-tool#51]
## Issue
There is no easy way for multi-signature Hedera accounts to send assets with a `memo`-info attached. 

## Potential solution
We should add support for a multi-sig Hedera account to utilize this tool to be able to assign a `memo`
 information on a collective basis (meaning, we ask for it once and it's being replicated to all resulting transactions) to all the individual-resulting transactions of the requested assets.

## Question
How does this manifest in the user journey?

Before final confirmation of the resulting assets distribution is asked for, the user is prompted to input the desired, _optional_, `memo` string which should be validated (current maximum size is 100 characters, I think).

**Comment by Cooper-Kunz** [comment]
Adding memo-support makes sense & is expected to come with #44 + v2 more generally. Currently it's expected to be supported in single transfers (see the screenshot in #50) and in the bulk transaction distribution by adding an additional column to the .csv input (so that way each tx in the bulk distribution can have a unique memo). Would that be sufficient?

**Comment by 3Nigma** [comment]
Yes. I think that would very much suffice. Thank you, @Cooper-Kunz !
